map_gui/src/render/pedestrian.rs

```diff
@@ -43,6 +43,25 @@ impl DrawPedestrian {
             );
         }
 
+        if input.intent == Some(Intent::SteepUphill) {
+            let bubble_z = -0.0001;
+            let mut bubble_batch =
+                GeomBatch::load_svg(prerender, "system/assets/map/thought_bubble.svg")
```

Comment:
I don't think these 3 bubbles are worth refactoring; the scale/translation parameters are tuned differently in each case anyway